components/rmrk/Gallery/EmotionList.vue

```diff
@@ -2,14 +2,14 @@
   <div
     v-show="emotes"
     class="columns is-mobile is-multiline is-gapless nft-emotion__main">
-    <div
-      v-for="emoji in emotes"
-      :key="emoji.key"
-      class="column nft-emotion__column">
-      <b-tag id="emoji-box" type="is-dark is-large">
+    <div v-for="emoji in emotes" :key="emoji.key" class="column">
+      <b-button
+        id="emoji-box"
+        type="is-dark"
+        @click="$emit('selected', emoji.key)">
         {{ String.fromCodePoint(parseInt(emoji.key, 16)) }}
         <span class="nft-emotion-tag__count">{{ emoji.count }}</span>
```

Comment:
```suggestion
        <span>{{ emoji.count }}</span>
```
I don't think we need the class for now, could be changed with spacing helper